### DISCUSSION TITLE
PnP search queries

### DISCUSSION THREAD

**Discussion by PradnyaKurane** [microsoft-search/pnp-modern-search#2432] [Q&A]
we have deployed pnp search web part in sharepoint modern site.
Facing below issues.
1. when searching for some document with keyword, showing all versions of that document. It should display 1 document as single entity
2. How to exclude preservation hold library documents. OR how to give libraries which need to include
3. column values are not geeting displayed

**Comment by PradnyaKurane** [comment]
<img width="501" alt="Issue1" src="https://user-images.githubusercontent.com/115466611/194851252-28d19562-8855-4e8f-b0b2-82f83ef60c4a.png">

**Reply by wobba** [reply]
You can limit to the libraries you want with e.g. `path:https://site.domain/sites/asite/shared%20documents/` You can exclude by using `NOT path:.....` 

This is one way.

**Comment by PradnyaKurane** [comment]
Hi team,
I have multiple subsites and multiple libraries . In this scenario also should we do the same?

Kind Regards,
Pradnya Kurane
Delivery Excellence | Process Excellence Team

 ***@***.*** <https://ind01.safelinks.protection.outlook.com/?url=http%3A%2F%2Fwww.lntinfotech.com%2F&data=04%7C01%7CPradnya.Kurane%40lntinfotech.com%7C091cb859fd77457e664408d9e546fa8a%7C02aa9fc118bc4798a020e01c854dd434%7C1%7C0%7C637792915289665130%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=aiOJN7v2YSbcD73hQVNdkFWBcz%2FgvisRnwkA%2Bh4G6JI%3D&reserved=0>
Larsen & Toubro Infotech Ltd.

**Reply by wobba** [reply]
Yes.
